repl/index.js

```diff
@@ -17,7 +17,7 @@ function wrap(name, optionsArgIndex) {
   vm[name] = function (code) {
     var options = arguments[optionsArgIndex];
     var filename = options && options.filename;
-    var args = [compile(code, filename)];
+    var args = [compile(code, filename, { generateLetDeclarations: false })];
     for (var i = 1; i < arguments.length; ++i) {
```

Comment:
Being able to set options for the REPL is probably a good thing regardless of the current nested import `let` issue.